Highcharts 8.1.1+ TS8010 error

I recently upgraded to Highcharts 8.1.1 to fix a security vulnerability. After this, I was unable to compile my angular application. Version 8.1.0 and below compile with no issues, but anything 8.1.1 or greater will not compile.

#### Expected behaviour
Application compiles

#### Actual behaviour
```
ERROR in node_modules/highcharts/highcharts.d.ts(781,51): error TS8020: JSDoc types can only be used inside documentation comments.
node_modules/highcharts/highcharts.d.ts(781,60): error TS8020: JSDoc types can only be used inside documentation comments.
node_modules/highcharts/highcharts.d.ts(781,69): error TS8020: JSDoc types can only be used inside documentation comments.
node_modules/highcharts/highcharts.d.ts(781,78): error TS8020: JSDoc types can only be used inside documentation comments.
node_modules/highcharts/highcharts.d.ts(781,87): error TS8020: JSDoc types can only be used inside documentation comments.
node_modules/highcharts/highcharts.d.ts(781,96): error TS8020: JSDoc types can only be used inside documentation comments.
node_modules/highcharts/highcharts.d.ts(781,105): error TS8020: JSDoc types can only be used inside documentation comments.
```

#### Product version
Highcharts: `8.1.1+`
`8.1.0` and below will compile
Typescript: `^2.9.2`

#### Affected browser(s)
All

--- a/js/parts/SVGRenderer.js
+++ b/js/parts/SVGRenderer.js
@@ -147,7 +147,7 @@ var addEvent = U.addEvent, attr = U.attr, createElement = U.createElement, css =
  * Array of path commands, that will go into the `d` attribute of an SVG
  * element.
  *
- * @typedef {Array<Array<Highcharts.SVGPathCommand,number?,number?,number?,number?,number?,number?,number?>>} Highcharts.SVGPathArray
+ * @typedef {Array<(Array<Highcharts.SVGPathCommand>|Array<Highcharts.SVGPathCommand,number>|Array<Highcharts.SVGPathCommand,number,number>|Array<Highcharts.SVGPathCommand,number,number,number,number>|Array<Highcharts.SVGPathCommand,number,number,number,number,number,number>|Array<Highcharts.SVGPathCommand,number,number,number,number,number,number,number>)>} Highcharts.SVGPathArray
  */
 /**
  * Possible path commands in an SVG path array. Valid values are `A`, `C`, `H`,

--- a/ts/parts/SVGRenderer.ts
+++ b/ts/parts/SVGRenderer.ts
@@ -412,7 +412,7 @@ declare global {
  * Array of path commands, that will go into the `d` attribute of an SVG
  * element.
  *
- * @typedef {Array<Array<Highcharts.SVGPathCommand,number?,number?,number?,number?,number?,number?,number?>>} Highcharts.SVGPathArray
+ * @typedef {Array<(Array<Highcharts.SVGPathCommand>|Array<Highcharts.SVGPathCommand,number>|Array<Highcharts.SVGPathCommand,number,number>|Array<Highcharts.SVGPathCommand,number,number,number,number>|Array<Highcharts.SVGPathCommand,number,number,number,number,number,number>|Array<Highcharts.SVGPathCommand,number,number,number,number,number,number,number>)>} Highcharts.SVGPathArray
  */
 
 /**

